### PR TITLE
Sol003 driver- Drivers vulnerabilities fixes

### PR DESCRIPTION
**Pull request by rkumar8j**
Issue: #41
 Description of the changes: #41
 List of related PRs. : None
 Project builds without any errors.
 Unit Tests updated (or created where needed) and all pass.
 You have ran manual functional “smoke” test (does product as a whole still work?).
 User Story meets the acceptance criteria - and passes. acceptance criteria should be understood at outset.
 There is a PR for update of internal documentation for the affected Project(s) - README etc.

### CODE REVIEW

**Comment by lokanalla** [comment]
Hi @hhellyer , @owen-lynch-ibm , Could you please review this PR for the vulnerabilities fix.

**Review by owen-lynch-ibm — APPROVED** [review]

File: src/main/resources/docker/Dockerfile [hunk 1]
```diff
@@ -11,6 +11,8 @@ RUN groupadd -g 1000 -r sld \
  && chown -R sld:0 /home/sld /var/lm /data \
  && chmod -R 775 /home/sld \
  && chmod -R 775 /var/lm
+RUN apt update -y \
+ && apt upgrade -y
```
Comment (hhellyer) [inline src/main/resources/docker/Dockerfile]: Running apt commands without cleaning up afterwards can inflate the image size significantly. You should clean up by running `rm -rf /var/lib/apt/lists/*` as the last step in that run command to make sure the image doesn't contain the apt cache.

See: https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run for more reference and sample docker commands to do this.

**Review by hhellyer — CHANGES_REQUESTED** [review]
The apt cache should be cleaned up now you are running `apt` commands to avoid increasing the image size more than is neccessary.

File: src/main/resources/docker/Dockerfile (hunk 1 above)
Comment (rkumar8j) [reply src/main/resources/docker/Dockerfile]: @hhellyer : Done the changes as per your review comment. Kindly review the same and let us know if any further code modification is needed.

**Review by hhellyer — APPROVED** [review]
Approved, thanks for making that update.

**Comment by rkumar8j** [comment]
Hi @hhellyer : Could you please merge the changes and close this PR.

**Pull request merged**